build.gradle

```diff
@@ -142,19 +243,28 @@ subprojects {
                     includeGroup("dev.ftb.mods")
                 }
             }
-
             maven {
-                name = "Forge Config Api Port"
-                url = "https://raw.githubusercontent.com/Fuzss/modresources/main/maven/"
-            }
-            maven { // Hexcasting
+                name = "Hexcasting, Paucal, Patchouli"
                 url = "https://maven.blamejared.com"
+                content {
+                    includeGroup("at.petra-k.hexcasting")
+                    includeGroup("at.petra-k.paucal")
+                    includeGroup("vazkii.patchouli")
+                }
             }
-            maven { // Hexical via temporary Maven
+            maven {
+                name = "[Temporary] Hexical"
                 url = "https://maven.pool.net.eu.org/"
+                content {
+                    includeGroup("miyucomics.hexical")
+                }
             }
-            maven { // Hexal
+            maven {
+                name = "Hexal"
                 url = uri("https://maven.hexxy.media")
+                content {
+                    includeGroup("ram.talia.hexal")
+                }
             }
         }
         maven {
```

Comment:
I'd also suggest put VS maven before `if (!project.block_external_repositories)`, so if a dependency is loaded in the VS maven it can be used first.